src/ExposureContext.tsx

```diff
@@ -68,20 +71,15 @@ const ExposureProvider: FunctionComponent = ({ children }) => {
     })
   }, [getLastDetectionDate])
 
-  useEffect(() => {
-    const handleAppStateChange = async () => {
-      const exposureInfos = await getCurrentExposures()
-      setExposureInfo(exposureInfos)
-    }
-
-    AppState.addEventListener("change", handleAppStateChange)
+  const refreshExposureInfo = async () => {
+    const exposureInfos = await getCurrentExposures()
+    setExposureInfo(exposureInfos)
 
-    handleAppStateChange()
+    const detectionDate = await getLastDetectionDate()
+    setLastExposureDetectionDate(detectionDate)
 
-    return () => {
-      AppState.removeEventListener("change", handleAppStateChange)
-    }
-  }, [])
+    return Promise.resolve()
```

Comment:
I don't think we need to return a response here, doesn't the `async` `await` calms the type signature?

Reply:
I removed it, thanks!